understanding/20/language-of-parts.html

```diff
@@ -13,7 +13,7 @@ <h2>Intent of Language of Parts</h2>
       
       
       <p>The intent of this Success Criterion is to ensure that user agents can correctly present
-         content written in multiple languages. This makes it possible for user agents and
+         words, phrases, or passages written in multiple languages. This makes it possible for user agents and
```

Comment:
Agree with @WilcoFiers 's point in today's meeting that it's problematic for this to be more strict than the normative wording of the SC, which is just "passage or phrase" rather than "words, phrases, or passages". I think this wording should match the normative SC wording.

Reply:
To me, the idea of "words" being applicable is part of the normative text

> except for proper names, technical terms, words of indeterminate language, and words or phrases that have become part of the vernacular of the immediately surrounding text.

As well, there is substantial existing wording in the Understanding document about words, especially the following sentence:
> Because this is so common in some languages, single words should be considered part of the language of the surrounding text unless it is clear that a change in language was intended. 

 These things, together, make it clear to me that replacing "content" with "words, phrases, or passages"  in the _non-normative_ text intro is accurate, more clear, and supported by the existing normative and non-normative language.

Reply:
Adding "words" to "passages and phrases" is not supported by the normative text IMO. Mike, that you need to point to the exception says everything to me. If the intent was for this to include every individual word, the success criterion would have said so explicitly. 

There are good reasons not to be this strict. It's really hard to decide consistently what words are part of jargon or idioms. This is a problem in places that tend to be multilingual like The Netherlands and Denmark. Most people here speak English, and because of our exposure to English media there are English words that are more familiar to people than their native counterparts.

Switching back and forth between speech synthesizers in the middle of a sentence can make that sentence far more difficult to understand than having a word mispronounced slightly. If your screen reader has a similar-sounding voice for both languages, that's less problematic than if it uses two completely different voices. It also matters how close the two languages are. Having an German word pronounced with a Dutch voice makes almost no difference.

IMO, individual words in a sentence should not be required to have a lang attribute. Doing so is likely to hinder the comprehension of that text, it's difficult to evaluate consistently, and it's a significant burden on content authors to boot.

Reply:
"there are English words that are more familiar to people than their native counterparts." Then those meet the exception.

Reply:
As discussed in the meeting, a [phrase can be a word](https://en.wikipedia.org/wiki/Phrase), but many people haven't understood that. That doesn't mean any word, but some. [Anyway, approved in the meeting](https://www.w3.org/2023/05/23-ag-minutes#item10).